src/components/UserProfileCard/UserProfileCardStyle.js

```diff
@@ -0,0 +1,17 @@
+import { makeStyles } from '@mui/styles';
+
+export const useStyles = makeStyles({
+  pos: {
+    color: '#112025',
+    fontSize: '20px',
+  },
+  box: {
+    borderRadius: 3,
+    boxShadow: '2px 2px 10px #33333333',
```

Comment:
Maybe it would be better to add such styles? 
`    boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.2)',
    borderRadius: '8px',`
Then it will look like input. What do you think?
![Zrzut ekranu 2022-02-09 o 21 20 45](https://user-images.githubusercontent.com/75137091/153283856-cdcbe5eb-49eb-4f80-9333-14676628fb18.png)

Reply:
fix